test/Microsoft.TestPlatform.AcceptanceTests/DisableAppdomainTests.cs

```diff
@@ -26,7 +26,7 @@ public void DisableAppdomainTest(RunnerInfo runnerInfo)
         var diableAppdomainTest1 = _testEnvironment.GetTestAsset("DisableAppdomainTest1.dll", "net451");
         var diableAppdomainTest2 = _testEnvironment.GetTestAsset("DisableAppdomainTest2.dll", "net451");
 
-        RunTests(runnerInfo.RunnerFramework, string.Format("{0}\" \"{1}", diableAppdomainTest1, diableAppdomainTest2), 2);
+        RunTests(runnerInfo, string.Format("{0}\" \"{1}", diableAppdomainTest1, diableAppdomainTest2), 2);
```

Comment:
Not related to your change but as the line is touched maybe we could replace the string format by a string interpolation?

Reply:
Won't fix, I am not trying to fix all the code here.